requirements.txt

```diff
@@ -1,2 +1,2 @@
-strawberryfields>=0.9
-pennylane>=0.4
+strawberryfields
```

Comment:
no versions?

Reply:
You're right, this is an oversight on my end.
```suggestion
strawberryfields>=0.11
```

Reply:
Ah, I remember now. Travis is set to install SF from github master - setting >=0.11 before 0.11 was released was causing pip to fail, so I temporarily removed version pinning.